### PR TITLE
feat(kubernetes): rename option

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
Close MBP-218

### Requirements

Add rename option for kubernetes

### Rename Kubernetes

Rename functionality has been added for Kubernetes

### CODE REVIEW

File: client/app/kubernetes/service/rename/kubernetes-service-rename.modal.controller.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    this.ControllerHelper = ControllerHelper;
+    this.openModal();
+  }
+
+  openModal() {
+    this.ControllerHelper.modal.showModal({
+      modalConfig: {
+        templateUrl: 'app/kubernetes/service/rename/kubernetes-service-rename.html',
+        controller: 'kubernetesRenameCtrl',
+        controllerAs: 'ctrl',
```
Comment (marie-j) [inline client/app/kubernetes/service/rename/kubernetes-service-rename.modal.controller.js]:
```suggestion
         controllerAs: '$ctrl',
```

File: client/app/kubernetes/service/rename/kubernetes-service-rename.modal.controller.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        backdrop: 'static',
+      },
+    })
+      .then(() => {
+        this.$scope.$parent.$parent.$parent.$ctrl.getCluster();
+        this.$scope.$parent.$ctrl.getClusterInfos();
```
Comment (marie-j) [inline client/app/kubernetes/service/rename/kubernetes-service-rename.modal.controller.js]: Could an event be used instead ? Or a callback on modal closing ?

File: client/app/kubernetes/kubernetes.html [hunk 3]
```diff
@@ -2,16 +2,22 @@
     <oui-spinner data-size="m"></oui-spinner>
 </div>
 <div data-ng-if="!$ctrl.loading">
-    <oui-page-header data-heading="{{:: $ctrl.cluster.name }}" data-description="{{:: $ctrl.cluster.id }}">
-        <oui-guide-menu data-text="{{:: 'kube_guides' | translate }}">
-            <oui-guide-menu-item data-text="{{:: 'kube_guides_kubernetes' | translate }}" data-href="{{:: $ctrl.KUBERNETES.urls.kubernetesDoc }}" data-external></oui-guide-menu-item>
-        </oui-guide-menu>
+    <header class="oui-page-header">
```
Comment (antleblanc) [inline client/app/kubernetes/kubernetes.html]: Could you tell why you are replacing the `<oui-page-header>` component?

File: client/app/kubernetes/service/kubernetes-service.html [hunk 4]
```diff
@@ -12,6 +12,18 @@
         <!--Cluster Information-->
         <div class="col-md-4 col-xm-12" data-ng-if="$ctrl.cluster">
             <oui-tile class="h-100" data-heading="{{:: 'kube_service_cluster_information' | translate }}">
+
+                <oui-tile-definition data-term="{{:: 'kube_service_name' | translate }}">
+                    <oui-tile-description data-ng-bind="$ctrl.cluster.name"></oui-tile-description>
+                    <oui-action-menu data-compact data-align="end">
+                        <oui-action-menu-item
+                            data-text="{{::'common_edit'|translate}}"
```
Comment (antleblanc) [inline client/app/kubernetes/service/kubernetes-service.html]: FYI `data-text` attribut is deprecated (ref: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/action-menu#deprecated)

File: client/app/kubernetes/service/kubernetes-service.html [hunk 5]
```diff
@@ -23,7 +35,7 @@
                               data-translate="{{ 'kube_service_cluster_status_' + $ctrl.cluster.status }}">
                         </span>
                     </oui-tile-description>
-                    <oui-action-menu data-compact align="end" data-disabled="$ctrl.cluster.status !== 'READY'"> 
+                    <oui-action-menu data-compact align="end" data-disabled="$ctrl.cluster.status !== 'READY'">
```
Comment (antleblanc) [inline client/app/kubernetes/service/kubernetes-service.html]:
```suggestion
                    <oui-action-menu data-compact data-align="end" data-disabled="$ctrl.cluster.status !== 'READY'">
```

File: client/app/kubernetes/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -29,6 +30,10 @@
   "kube_service_reset_success": "Votre cluster Kubernetes est en cours de réinitialisation.",
   "kube_service_reset_error": "Une erreur est survenue lors de la réinitialisation de votre cluster : {{ message }}",
   "kube_service_reset_enter": "Entrez \"RESET\" dans le champ ci-dessous pour confirmer",
+  "kube_service_rename_title": "Modifier le nom",
+  "kube_service_rename_description": "Modifier le nom du service kubernetes",
+  "kube_service_rename_success": "Le nom a été modifié avec succès",
+  "kube_service_rename_error": "Une erreur est survenue lors de la modification du nom: {{message}}",
```
Comment (antleblanc) [inline client/app/kubernetes/translations/Messages_fr_FR.json]:
```suggestion
  "kube_service_rename_error": "Une erreur est survenue lors de la modification du nom : {{message}}",
```

**Pull request merged**